### ISSUE TITLE
GSLB: Unable to create A and AAAA wide-ip's using the same domain name

### ISSUE THREAD

**Issue by LeonardosGitHub** [F5Networks/f5-appsvcs-extension#167]
##### Do you already have an issue opened with F5 support?
Github Issues are consistently monitored by F5 staff, but should be considered as best effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open an case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

##### ISSUE TYPE

 - Bug Report

##### AS3 BUILD/ VERSION

```
3.12.0-5
```

##### BIGIP VERSION

```
14.0.0.1
```

##### SUMMARY

```
GSLB: Unable to create "A" and "AAAA" wide-ip's using same domain name.
```

##### STEPS TO REPRODUCE

```
{
    "class": "ADC",
    "schemaVersion": "3.11.0",
    "Common": {
        "class": "Tenant",
        "Shared": {
            "class": "Application",
            "template": "shared",
            "WIP_AAAA_domain": {
                "class": "GSLB_Domain",
                "resourceRecordType": "AAAA",
                "domainName": "wip.example.com"
            },
            "WIP_A_domain": {
                "class": "GSLB_Domain",
                "resourceRecordType": "A",
                "domainName": "wip.example.com"
            }
        }
    }
}
```

##### EXPECTED RESULTS

```
I expect to see 2 Wide-IPs when browsing to GUI > DNS > GSLB > Wide-IP > Wide-IP list:

1) wip.example.com Type: A
2) wip.example.com Type: AAAA

```

##### ACTUAL RESULTS

```
I only see 1 Wide-IPs when browsing to GUI > DNS > GSLB > Wide-IP > Wide-IP list:

1) wip.example.com Type: A
```

**Comment by dstokesf5** [comment]
Thank you. I have confirmed the bug and created internal product backlog item AUTOTOOL-860 to track it.

**Comment by dstokesf5** [comment]
It looks like this was resolved in 3.16.0. Closing this issue. If you are still experiencing a problem, please reopen this issue or create a new one.
